aioautomower/session.py

```diff
@@ -76,6 +76,11 @@ async def connect(self):
         if self.token is None:
             _LOGGER.debug("No token to connect with.")
             return False
+        if "amc:api" not in self.token["scope"]:
```

Comment:
You can even put the self.ws_task =... in an else clause, since we dont need to start it if the key doesnt support it